src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0 starline">
+      <div class="stars__star-inactive"></div>
```

Comment:
by default every start is inactive, so we don't need another class to explicitly set that